stdlib/contrib/rhajek/bigpanda/bigpanda.flux

```diff
@@ -27,12 +82,54 @@ statusFromLevel = (level) => {
     return sev
 }
 
-// `sendAlert` sends a single alert to BigPanda as described in https://docs.bigpanda.io/reference#alerts API. 
-// `token` - string - BigPanda authorization Bearer token
-// `url` - string - base URL of [BigPanda API](https://docs.bigpanda.io/reference#alerts).
-// `appKey` - string - BigPanda App Key.
-// `status` - string - Status of the BigPanda alert. One of ok, critical, warning, acknowledged.
-// `rec` - record - additional data appended to alert
+// sendAlert sends an alert to [BigPanda](https://www.bigpanda.io/).
+//
+// ## Parameters
+// - url: BigPanda [alerts API URL](https://docs.bigpanda.io/reference#alerts-how-it-works).
+//   Default is the value of the `bigpanda.defaultURL` option.
+// - token: BigPanda [API Authorization token (API key)](https://docs.bigpanda.io/docs/api-key-management).
+// - appKey: BigPanda [App Key](https://docs.bigpanda.io/reference#integrating-monitoring-systems).
+// - status: BigPanda [alert status](https://docs.bigpanda.io/reference#alerts).
+//   Supported statuses:
```

Comment:
Without an empty newline here, the markdown parser sees these as one paragraph. Because the first paragraph is used as the short description, we want to separate out the supported values into only the long description.

```suggestion
// - status: BigPanda [alert status](https://docs.bigpanda.io/reference#alerts).
//
//   Supported statuses:
```